src/ingest_llm_as/processors/conversation_ingestor.py

```
@@ -8,19 +8,22 @@
+from ingest_llm_as.models.knowledge_digest import create_conversation_digest
 
 logger = logging.getLogger(__name__)
 
 class ConversationIngestor:
     """
-    Polls 'raw_conversations' table, summarizes content, and ingests into Vault & Graph.
+    Polls 'raw_conversations' table, summarizes content, and sends to OmegaKG validation API.
+    
+    REFACTORED: No longer writes to Neo4j/pgvector directly.
     """
     def __init__(self):
```

Ruff (0.14.11)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)